# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: netwerk/test/unit/test_304_responses.js
# Commit: 74f99033251c
# Full Hash: 74f99033251cd6c36eb6b0b5a4713eb6b5c793fa
# Author: Jonathan Kingston <jkt@mozilla.com>
# Date: 2019-02-13 09:48:26
# Regressor Bug: 1520868
# File Overlap Count: 1
# Description:
#   Bug 1520868 - Replacing AsyncOpen2 with AsyncOpen always r=valentin
#   
#   Replacing js and text occurences of asyncOpen2
#   Replacing open2 with open
#   
# ==============================================================================

diff -r 3378fd5f4f22 -r 74f99033251c netwerk/test/unit/test_304_responses.js
--- a/netwerk/test/unit/test_304_responses.js	Tue Feb 12 17:58:19 2019 +0100
+++ b/netwerk/test/unit/test_304_responses.js	Tue Feb 12 16:08:25 2019 +0000
@@ -68,7 +68,7 @@
 // a 304 response (i.e. when the server shouldn't have sent us one).
 add_test(function test_unexpected_304() {
   var chan = make_channel(baseURI + unexpected304);
-  chan.asyncOpen2(new ChannelListener(consume304, null));
+  chan.asyncOpen(new ChannelListener(consume304, null));
 });
 
 // Test that we can cope with a 304 response that was (erroneously) stored in
@@ -90,6 +90,6 @@
       chan.QueryInterface(Ci.nsIHttpChannel);
       chan.setRequestHeader("If-None-Match", '"foo"', false);
 
-      chan.asyncOpen2(new ChannelListener(consume304, null));
+      chan.asyncOpen(new ChannelListener(consume304, null));
     });
 });